src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/idam/IdamTestService.java

```diff
@@ -58,37 +69,66 @@ public IdamTestService(
 
     public User createSolicitor() {
         String email = testData.nextUserEmail();
-        createUser(createSolicitorRequest(email, aatConfiguration.getSmokeTestSolicitor().getPassword()));
-        return userService.authenticateUser(email, aatConfiguration.getSmokeTestSolicitor().getPassword());
+        return Failsafe.with(retryPolicy)
+            .get(() -> {
+                try {
+                    idamTestApi.createUser(
+                        createSolicitorRequest(email, aatConfiguration.getSmokeTestCitizen().getPassword())
```

Comment:
Should that be `getSmokeTestSolicitor()` instead?

Reply:
nope - this is ok - shared pw